src/composablesTs/useBreakpoint.ts

```diff
@@ -3,32 +3,34 @@ import { isSSR } from "../utilsTs/utilHelper.ts";
 
 import type { Ref } from "vue";
 
-const BREAKPOINT_NAME = {
-  xs: "xs",
-  sm: "sm",
-  md: "md",
-  lg: "lg",
-  xl: "xl",
-  "2xl": "2xl",
-};
-
-const BREAKPOINT = {
-  xs: 0,
-  sm: 640,
-  md: 768,
-  lg: 1024,
-  xl: 1280,
-  "2xl": 1536,
-};
+enum BREAKPOINT_NAME {
```

Comment:
follow ts name convention for enums: PascalCase and for enum keys PascalCase
https://aykhanhuseyn.medium.com/typescript-naming-conventions-crafting-maintainable-code-7d872234fe17

in this case keys is better to be in UPPER_CASE: `BreakpointName.XS`